app/controllers/api/graphql_controller.rb

```diff
@@ -2,6 +2,9 @@
 
 module Api
   class GraphqlController < ApplicationController
+    skip_before_action :validate_request
```

Comment:
@bdunne We still haven't included the mixins needed for the validate request because of spec failures

https://github.com/ManageIQ/sources-api/blob/0c096f39ee826b8269e79a28bc01475183ae2475/app/controllers/application_controller.rb#L2

Does the skip_before_action check that the method is defined.

Also is this going to conflict with PR https://github.com/ManageIQ/manageiq-api-common/pull/91